test/DebugInfo/basic.swift

```diff
@@ -1,5 +1,3 @@
-// FIXME(integers): started to fail due to the new integer protocols
-// XFAIL: *
```

Comment:
There was a proposal to teach lit about test annotations so that anything `XFAIL`ed would have at least a PR associated. I think lldb used it for a bit, but I'm not sure if the idea has ever gotten traction. I think it would be useful both for LLVM (the whole umbrella), and Swift.